### PR TITLE
update gradle plugin repositories

### PR DESCRIPTION
**Pull request by ravisingal**
Fixes Github action failure:
```
FAILURE: Build failed with an exception.
* Where:
Settings file '/home/runner/work/db-schema-manager/db-schema-manager/settings.gradle.kts' line: 4
* What went wrong:
null cannot be cast to non-null type kotlin.String
```

https://github.com/hypertrace/db-schema-manager/runs/7455376000?check_suite_focus=true

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+    gradlePluginPortal()
+    maven("https://hypertrace.jfrog.io/artifactory/maven")
   }
 }
 
 plugins {
   id("org.hypertrace.version-settings") version "0.2.0"
 }
+
+rootProject.name = "db-schema-manager"
```
Comment (aaron-steinfeld) [inline settings.gradle.kts]:
```suggestion
rootProject.name = "db-schema-manager-root"
```

We've stopped naming root projects to match their repo name as it breaks the gradle type safe accessors

**Review by aaron-steinfeld — APPROVED** [review]

**Comment by aaron-steinfeld** [comment]
Ah crap - was trying to dismiss my review. The fact the build fails shows the CI is misconfigured

**Comment by aaron-steinfeld** [comment]
It's using pull_request_target as its trigger, which we need to when referencing secrets, but since we're not here we can change it to `pull_request`. That will allow the changes of this PR to be validated, rather than the code on main which is what just ran here.

If we were referencing secrets, we'd maintain the pull_request_target and change the checkout refs so that the workflow can't be changed in a pull request, but that it still runs against the PR's code. E.g. https://github.com/hypertrace/attribute-service/blob/705c5b4c2afdb7a745dfc30c9727e269011d3ade/.github/workflows/pr-build.yml#L17-L20

**Comment by ravisingal** [comment]
> It's using pull_request_target as its trigger, which we need to when referencing secrets, but since we're not here we can change it to `pull_request`. That will allow the changes of this PR to be validated, rather than the code on main which is what just ran here.
> 
> If we were referencing secrets, we'd maintain the pull_request_target and change the checkout refs so that the workflow can't be changed in a pull request, but that it still runs against the PR's code. E.g. https://github.com/hypertrace/attribute-service/blob/705c5b4c2afdb7a745dfc30c9727e269011d3ade/.github/workflows/pr-build.yml#L17-L20

Changed it in another PR: https://github.com/hypertrace/db-schema-manager/pull/3

**Pull request merged**